3604/1/get_information_about_gem.rb

```diff
@@ -0,0 +1,89 @@
+require 'mechanize'
+
+# class for get github url something gem and get information about this gem
+class GetInformationAboutGem
+  attr_reader :temporary_array
+
+  def initialize(name_gem)
+    @name_gem = name_gem
+    @url = 'https://rubygems.org/'
+    @agent = Mechanize.new
+    @temporary_array = []
+  end
+
+  def information_about_gem
+    if such_gem?
+      entry_all_values
+      temporary_array
```

Comment:
why don't u just build the result 
`[name, forks, score, whatever]`
and use hashes here